DashWallet/Sources/Models/Taxes/Tx/TxUserInfo.swift

```diff
@@ -109,3 +144,9 @@ extension DSTransaction {
         return category.stringValue
     }
 }
+
+func pow(_ base:Int, _ power:Int) -> Int {
+    var answer = 1
+    for _ in 0..<power { answer *= base }
+    return answer
+}
```

Comment:
Isn't there `pow` function in Swift?

Reply:
@Syn-McJ Swift std provides `pow` function, but only for floating-point value types. Since we work with `Int` here, it will be faster to use own version instead of converting it first to Float or Double.